src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition-duration: 0.3s;
```

Comment:
FYI: it's always better (for performance) to define transition-property (as you did in catalog.scss)